website/pages/docs/nia/architecture.mdx

```diff
@@ -6,4 +6,38 @@ description: >-
   Consul-Terraform-Sync Architecture
 ---
 
-# Consul-Terraform-Sync Architecture
+# Consul Terraform Sync Architecture
+
+Consul Terraform Sync is a service-oriented tool for managing
+network infrastructure near real-time. Consul-Terraform-Sync runs as a daemon
+and integrates the network topology maintained by your Consul cluster with your
+network infrastructure to dynamically secure and connect services.
+
+## 10,000 Foot View
+
+[![Consul Terraform Sync Architecture](/img/nia-highlevel-diagram.png)](/img/nia-highlevel-diagram.png)
+
+The diagram shows Consul Terraform Sync monitoring the Consul service catalog
+for updates and utilizing Terraform to update the state of the infrastructure.
+
+There are two principal aspects of Sync to know about corresponding to the
+lines to Consul and Terraform in the diagram above. The line to Consul
+represents the Watches monitoring the state of Consul's service catalog (and
```

Comment:
for "Watches" here, just want to double check if it should be "Watcher" so that it matches the header below?

Reply:
Thanks. I waffled back and forth between the 2 and decided to stick with what was in the code... but I missed this one.